src/components/shared/layout/Filterbar.vue

```diff
@@ -31,6 +33,14 @@
                 type: String,
                 default: '',
             },
+
+            /**
+            * Show search
+            */
+            showSearch: {
```

Comment:
If we are wrapping this in a slot, we wouldn't need this prop.

Reply:
I was thinking this, but then what if we want the left hand side to be empty? we would need to add an empty div? What do you think?

Reply:
I don't know, personally I don't understand why we would want to use this without the only real element in the component... without the search bar, it is basically a secondary segment.